components/elm/src/main/elmfates_interfaceMod.F90

```diff
@@ -464,17 +469,23 @@ subroutine ELMFatesGlobals2()
            pass_logging = 0
         end if
 
-!        if(do_elm_fates_harvest) then
         if(get_do_harvest()) then
            pass_logging = 1
            pass_num_lu_harvest_types = num_harvest_vars
            pass_lu_harvest = 1
+           if(allow_harvest_bypass_criteria) then
+              pass_harvest_bypass_criteria = 1
+           else
+              pass_harvest_bypass_criteria = 0
+           end if
         else
            pass_lu_harvest = 0
            pass_num_lu_harvest_types = 0
+           pass_harvest_bypass_criteria = 0
         end if
 
         call set_fates_ctrlparms('use_lu_harvest',ival=pass_lu_harvest)
+        call set_fates_ctrlparms('use_harvest_bypass_criteria',ival=pass_harvest_bypass_criteria)
```

Comment:
I don't see any setup for this on the fates side pr 888.  Is this a future feature?  I'm assuming so given that it's hard coded to false just above this.

Reply:
It's an old flag. I forgot to delete this line in ELM side.